packages/boot/test/bootstrapTests/test-vat-orchestration.ts

```diff
@@ -70,3 +70,20 @@ test('createAccount returns an ICA connection', async t => {
     accountAddress,
   });
 });
+
+test('ICA connection can be closed', async t => {
```

Comment:
Is it important that closing a connection frees up resources? If so, is it feasible to check that?

I can imagine it might not be feasible... that any relevant collections are weak / non-enumerable.

Reply:
These are good questions. I created #9192 to track